public/app/features/dashboard-scene/scene/DashboardSceneRenderer.tsx

```diff
@@ -44,10 +45,18 @@ export function DashboardSceneRenderer({ model }: SceneComponentProps<DashboardS
       {editPanel && <editPanel.Component model={editPanel} />}
       {!editPanel && (
         <CustomScrollbar autoHeightMin={'100%'}>
-          <div className={styles.canvasContent}>
-            <NavToolbarActions dashboard={model} />
-            {controls && <controls.Component model={controls} />}
-            {isEmpty ? emptyState : withPanels}
+          <div className={styles.innerScrollbarContainer}>
+            {scopeSelector && <scopeSelector.Component model={scopeSelector} renderExpanded />}
```

Comment:
The scope selector will also make sense in the edit mode as one can execute queries there. And those queries need to respect the selected scope. However, I think we may want to consider it beind either hidden or in a readonly mode.

Reply:
Also - I don't think the ScopeSelector should be rendered within the dashboard's scroll area. This results in the following UX:


https://github.com/grafana/grafana/assets/2376619/46cfa7aa-42bf-497b-a0c7-b24abfbeedb2

IMO the scope selector pane should have inner scrollbar that allows dashboard list scrolling, and the dashboard should have it's own scroll.